### DISCUSSION TITLE
Azuracast and problems with song titles

### DISCUSSION THREAD

**Discussion by lotussutol** [AzuraCast/AzuraCast#6202] [Q&A] [answered]
Anybody else have this problem and if so has anybody been able to fix it?
The embed player does not show special characters correctly and if the title has a - in it...the song title stops at that point. 
See attachments for both problems.
![1](https://user-images.githubusercontent.com/5403151/230788166-f6a6fdb8-9e5f-4edd-b0be-287b9a39257f.PNG)
![4](https://user-images.githubusercontent.com/5403151/230788167-2a24c0a0-d304-4ac2-ae5a-bbd3a060786b.PNG)
![3](https://user-images.githubusercontent.com/5403151/230788168-b988c074-cd3d-4e28-ab61-c233f6c705d5.PNG)
![2](https://user-images.githubusercontent.com/5403151/230788169-25bbc53a-fcd4-469d-90e2-470bb62c236a.PNG)

**Comment by scysys** [comment, marked as answer]
Under station settings check your character encoding and set it to UTF-8

**Reply by lotussutol** [reply]
Thanks! I did find this. It was more than one place too. Meta Data starts with the broadcasting software, then to the encoder software, and then to Azuracast. So, any future people having this problem make sure you set it to UTF-8 in all the places touches the information.
